### PR TITLE
Add `user-select: none` to several parts of the debugger.

### PR DESCRIPTION
**Pull request by wldcordeiro**
Associated Issue: #841 

### Summary of Changes

* add `user-select: none` to various parts of the debugger so selection/ctrl+a only gets the things we expect (note, bare `user-select` seems to not work so adding the vendor prefixes for full support.)

### Testing

* [x] passes `npm test`
* [x] passes `npm run lint`

### CODE REVIEW

File: public/js/components/Editor.css [hunk 1]
```diff
@@ -90,4 +90,9 @@
   background-color: var(--theme-tab-toolbar-background);
   font-weight: lighter;
   z-index: 100;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  -o-user-select: none;
+  user-select: none;
```
Comment (jlongster) [inline public/js/components/Editor.css]: Does this apply it to the text within CodeMirror? The code should be selectable.

Comment (wldcordeiro) [inline public/js/components/Editor.css]: screenshot_20161006_133621

File: public/js/components/Breakpoints.css [hunk 2]
```diff
@@ -1,3 +1,10 @@
+.breakpoints-list * {
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  -o-user-select: none;
+  user-select: none;
```
Comment (jasonLaster) [inline public/js/components/Breakpoints.css]: we should only need `user-select`. prefixes are the worst

**Comment by wldcordeiro** [comment]
screenshot_20161006_135958

**Pull request merged**